src/legacy/ui/public/styles/_legacy/_base.scss

```diff
@@ -59,8 +59,16 @@ input[type="checkbox"],
     padding-bottom: $euiSizeS;
   }
 
-  > kbn-top-nav {
+  > kbn-top-nav, kbn-top-nav2 {
```

Comment:
Can this Sass be moved closer to the JS it touches. I don't know that this belongs in base.

All the edits below should then probably be prefixed with `.kbnTopNav` ..etc

Reply:
moved to directive folder \ new component folder